cef3/tools/configure.sh

```diff
@@ -12,8 +12,9 @@ TOOL_DIR="$( cd "$( dirname "$0" )" && pwd )"
 
 if [ ! -d "$GYP_DIR" ]; then
   echo "Checking out gyp into tools/gyp..."
-  svn checkout --force http://gyp.googlecode.com/svn/trunk "$GYP_DIR" \
-      --revision 1895
+  git clone https://chromium.googlesource.com/external/gyp "$GYP_DIR"
+#  svn checkout --force http://gyp.googlecode.com/svn/trunk "$GYP_DIR" \
+#      --revision 1895
```

Comment:
Thanks! Can you add the following to use the code at revision 1895 because newer gyp is not tested with this project? And remove the "svn checkout" lines.

```
pushd "$GYP_DIR"
git checkout 9a184eea4849278fd557627adcf48cb8e2ff9f67
popd
```